### PR TITLE
Create ShreyaansJain.json

### PR DESCRIPTION
**Pull request by shreyaansjain06**
Fixes #

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/234"></a>

### CODE REVIEW

File: public/data/ShreyaansJain.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (FayasNoushad) [inline public/data/ShreyaansJain.json:1]: Remove this line

**Review by FayasNoushad — APPROVED** [review]

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

**Comment by Cahllagerfeld** [comment]
Awesome! Thanks for your contribution 👍

File: public/data/ShreyaansJain.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "avatar": "https://github.com/shreyaansjain06.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/shreyaansjain06",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/shreyaansjain06",
```
Comment (Panquesito7) [inline public/data/ShreyaansJain.json:13]:
```suggestion
      "url": "https://twitter.com/shreyaansjain06",
```

File: public/data/ShreyaansJain.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Shreyaans Jain",
+  "bio": "Web development enthusiast | Ug at IIT Kanpur | Open Source Contributor | Focus on getting solution real life problem through Tech",
+  "avatar": "https://github.com/shreyaansjain06.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/shreyaansjain06",
```
Comment (Panquesito7) [inline public/data/ShreyaansJain.json:8]:
```suggestion
      "url": "https://github.com/shreyaansjain06",
```

**Pull request merged**